### ISSUE TITLE
Evaluation of the trained AO front-end model yielded poor results

### ISSUE THREAD

**Issue by qq2418916003** [LUMIA-Group/Leveraging-Self-Supervised-Learning-for-AVSR#6]
I trained AO's front-end model for 40 epochs and got a loss of 1.95 and a WER of 1.0, and WER has been around 1.0 since the beginning.

--------train---------
Epoch 41:   0%|                        | 0/183 [00:00<00:00, 3284.50it/s, loss=1.99, info/val_loss=1.950, info/val_WER=1.000, info/val_TF_WER=1.000, info/train_loss=2.000]
-------eval----------
Eval:   0%|                                         | 0/39 [00:00<?, ?it/s]batch1 || Test CER: 0.972 || Test WER: 0.972
2022-10-04 15:28:55 | INFO | __main__ | batch1 || Test CER: 0.972 || Test WER: 0.972
Eval:   3%|▊                                | 1/39 [00:52<33:11, 52.40s/it]batch2 || Test CER: 0.973 || Test WER: 0.976
2022-10-04 15:30:09 | INFO | __main__ | batch2 || Test CER: 0.973 || Test WER: 0.976
Eval:   5%|█▋                               | 2/39 [02:05<39:50, 64.60s/it]batch3 || Test CER: 0.970 || Test WER: 0.977
2022-10-04 15:30:36 | INFO | __main__ | batch3 || Test CER: 0.970 || Test WER: 0.977
Eval:   8%|██▌                              | 3/39 [02:32<28:27, 47.42s/it]batch4 || Test CER: 0.970 || Test WER: 0.979
2022-10-04 15:31:02 | INFO | __main__ | batch4 || Test CER: 0.970 || Test WER: 0.979
Eval:  10%|███▍                             | 4/39 [02:59<22:52, 39.21s/it]batch5 || Test CER: 0.971 || Test WER: 0.982
2022-10-04 15:31:29 | INFO | __main__ | batch5 || Test CER: 0.971 || Test WER: 0.982
Eval:  13%|████▏                            | 5/39 [03:26<19:41, 34.76s/it]batch6 || Test CER: 0.972 || Test WER: 0.984
2022-10-04 15:32:18 | INFO | __main__ | batch6 || Test CER: 0.972 || Test WER: 0.984
Eval:  15%|█████                            | 6/39 [04:14<21:42, 39.47s/it]batch7 || Test CER: 0.972 || Test WER: 0.984
2022-10-04 15:32:57 | INFO | __main__ | batch7 || Test CER: 0.972 || Test WER: 0.984
Eval:  18%|█████▉                           | 7/39 [04:53<21:01, 39.41s/it]batch8 || Test CER: 0.972 || Test WER: 0.986
2022-10-04 15:33:33 | INFO | __main__ | batch8 || Test CER: 0.972 || Test WER: 0.986
Eval:  21%|██████▊                          | 8/39 [05:29<19:46, 38.28s/it]batch9 || Test CER: 0.972 || Test WER: 0.986
2022-10-04 15:34:43 | INFO | __main__ | batch9 || Test CER: 0.972 || Test WER: 0.986
Eval:  23%|███████▌                         | 9/39 [06:40<24:09, 48.30s/it]batch10 || Test CER: 0.971 || Test WER: 0.986
2022-10-04 15:35:28 | INFO | __main__ | batch10 || Test CER: 0.971 || Test WER: 0.986
Eval:  26%|████████▏                       | 10/39 [07:24<22:46, 47.12s/it]batch11 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:36:27 | INFO | __main__ | batch11 || Test CER: 0.971 || Test WER: 0.984
Eval:  28%|█████████                       | 11/39 [08:24<23:46, 50.95s/it]batch12 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:37:14 | INFO | __main__ | batch12 || Test CER: 0.971 || Test WER: 0.984
Eval:  31%|█████████▊                      | 12/39 [09:11<22:23, 49.77s/it]batch13 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:37:41 | INFO | __main__ | batch13 || Test CER: 0.971 || Test WER: 0.984
Eval:  33%|██████████▋                     | 13/39 [09:38<18:35, 42.89s/it]batch14 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:38:24 | INFO | __main__ | batch14 || Test CER: 0.971 || Test WER: 0.984
Eval:  36%|███████████▍                    | 14/39 [10:20<17:47, 42.69s/it]batch15 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:38:53 | INFO | __main__ | batch15 || Test CER: 0.971 || Test WER: 0.984
Eval:  38%|████████████▎                   | 15/39 [10:49<15:27, 38.64s/it]batch16 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:39:29 | INFO | __main__ | batch16 || Test CER: 0.971 || Test WER: 0.984
Eval:  41%|█████████████▏                  | 16/39 [11:25<14:29, 37.79s/it]batch17 || Test CER: 0.971 || Test WER: 0.985
2022-10-04 15:40:22 | INFO | __main__ | batch17 || Test CER: 0.971 || Test WER: 0.985
Eval:  44%|█████████████▉                  | 17/39 [12:18<15:30, 42.31s/it]batch18 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:40:48 | INFO | __main__ | batch18 || Test CER: 0.971 || Test WER: 0.984
Eval:  46%|██████████████▊                 | 18/39 [12:45<13:09, 37.61s/it]batch19 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:41:15 | INFO | __main__ | batch19 || Test CER: 0.971 || Test WER: 0.983
Eval:  49%|███████████████▌                | 19/39 [13:11<11:25, 34.28s/it]batch20 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:41:59 | INFO | __main__ | batch20 || Test CER: 0.971 || Test WER: 0.983
Eval:  51%|████████████████▍               | 20/39 [13:55<11:48, 37.27s/it]batch21 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:42:46 | INFO | __main__ | batch21 || Test CER: 0.971 || Test WER: 0.984
Eval:  54%|█████████████████▏              | 21/39 [14:43<12:06, 40.36s/it]batch22 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:43:21 | INFO | __main__ | batch22 || Test CER: 0.971 || Test WER: 0.983
Eval:  56%|██████████████████              | 22/39 [15:17<10:56, 38.61s/it]batch23 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:44:02 | INFO | __main__ | batch23 || Test CER: 0.971 || Test WER: 0.984
Eval:  59%|██████████████████▊             | 23/39 [15:58<10:27, 39.21s/it]batch24 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:44:29 | INFO | __main__ | batch24 || Test CER: 0.971 || Test WER: 0.983
Eval:  62%|███████████████████▋            | 24/39 [16:25<08:53, 35.54s/it]batch25 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:45:03 | INFO | __main__ | batch25 || Test CER: 0.971 || Test WER: 0.983
Eval:  64%|████████████████████▌           | 25/39 [17:00<08:14, 35.31s/it]batch26 || Test CER: 0.971 || Test WER: 0.984
2022-10-04 15:45:30 | INFO | __main__ | batch26 || Test CER: 0.971 || Test WER: 0.984
Eval:  67%|█████████████████████▎          | 26/39 [17:27<07:06, 32.83s/it]batch27 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:45:58 | INFO | __main__ | batch27 || Test CER: 0.971 || Test WER: 0.983
Eval:  69%|██████████████████████▏         | 27/39 [17:54<06:15, 31.26s/it]batch28 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:46:46 | INFO | __main__ | batch28 || Test CER: 0.971 || Test WER: 0.983
Eval:  72%|██████████████████████▉         | 28/39 [18:42<06:37, 36.15s/it]batch29 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:47:13 | INFO | __main__ | batch29 || Test CER: 0.971 || Test WER: 0.983
Eval:  74%|███████████████████████▊        | 29/39 [19:09<05:34, 33.43s/it]batch30 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:47:52 | INFO | __main__ | batch30 || Test CER: 0.971 || Test WER: 0.983
Eval:  77%|████████████████████████▌       | 30/39 [19:48<05:16, 35.11s/it]batch31 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:48:19 | INFO | __main__ | batch31 || Test CER: 0.971 || Test WER: 0.983
Eval:  79%|█████████████████████████▍      | 31/39 [20:15<04:20, 32.61s/it]batch32 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:48:45 | INFO | __main__ | batch32 || Test CER: 0.971 || Test WER: 0.983
Eval:  82%|██████████████████████████▎     | 32/39 [20:42<03:35, 30.81s/it]batch33 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:49:15 | INFO | __main__ | batch33 || Test CER: 0.971 || Test WER: 0.983
Eval:  85%|███████████████████████████     | 33/39 [21:12<03:03, 30.62s/it]batch34 || Test CER: 0.971 || Test WER: 0.983
2022-10-04 15:50:25 | INFO | __main__ | batch34 || Test CER: 0.971 || Test WER: 0.983
Eval:  87%|███████████████████████████▉    | 34/39 [22:21<03:31, 42.31s/it]batch35 || Test CER: 0.970 || Test WER: 0.983
2022-10-04 15:51:13 | INFO | __main__ | batch35 || Test CER: 0.970 || Test WER: 0.983
Eval:  90%|████████████████████████████▋   | 35/39 [23:09<02:55, 43.97s/it]batch36 || Test CER: 0.970 || Test WER: 0.982
2022-10-04 15:51:43 | INFO | __main__ | batch36 || Test CER: 0.970 || Test WER: 0.982
Eval:  92%|█████████████████████████████▌  | 36/39 [23:39<01:59, 39.82s/it]batch37 || Test CER: 0.970 || Test WER: 0.982
2022-10-04 15:52:10 | INFO | __main__ | batch37 || Test CER: 0.970 || Test WER: 0.982
Eval:  95%|██████████████████████████████▎ | 37/39 [24:06<01:11, 35.91s/it]batch38 || Test CER: 0.970 || Test WER: 0.982
2022-10-04 15:52:41 | INFO | __main__ | batch38 || Test CER: 0.970 || Test WER: 0.982
Eval:  97%|███████████████████████████████▏| 38/39 [24:37<00:34, 34.42s/it]batch39 || Test CER: 0.970 || Test WER: 0.982
2022-10-04 15:53:11 | INFO | __main__ | batch39 || Test CER: 0.970 || Test WER: 0.982
AOMODAL || Test CER: 0.970 || Test WER: 0.982
2022-10-04 15:53:11 | INFO | __main__ | AOMODAL || Test CER: 0.970 || Test WER: 0.982
